Models/TerritoryStats.ts

```diff
@@ -0,0 +1,31 @@
+class TerritoryStats {
```

Comment:
This should be an interface.

Reply:
https://blog.logrocket.com/interfaces-in-typescript-what-are-they-and-how-do-we-use-them-befbc69b38b3/

Reply:
done